Gemfile.lock

```diff
@@ -138,49 +138,49 @@ GEM
     method_source (1.0.0)
     mime-types (3.3.1)
       mime-types-data (~> 3.2015)
-    mime-types-data (3.2020.0512)
+    mime-types-data (3.2020.1104)
     mimemagic (0.3.5)
-    mini_magick (4.10.1)
+    mini_magick (4.11.0)
     mini_mime (1.0.2)
     mini_portile2 (2.4.0)
-    minitest (5.14.1)
+    minitest (5.14.2)
     msgpack (1.3.3)
     netrc (0.11.0)
-    nio4r (2.5.2)
+    nio4r (2.5.4)
```

Comment:
### [nio4r](https://github.com/socketry/nio4r)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 2.5.2 → 2.5.4

[_[change-log](https://github.com/socketry/nio4r/blob/master/CHANGES.md), [source-code](https://github.com/socketry/nio4r/tree/v2.5.4)_]





<details>
<summary>Commits</summary>

A change of **50** commits. See the full changes on [the compare page](https://github.com/socketry/nio4r/compare/v2.5.2...v2.5.4).

These are the first 10 commits:
- (d3a23fb) [Update CHANGES.md](https://github.com/socketry/nio4r/commit/d3a23fbe9525299a6a25a9fe8b0ef30077bb2049)
- (72e492a) [Update CHANGES.md](https://github.com/socketry/nio4r/commit/72e492a33787841b14fa1ed3c2297cd8c8fed367)
- (7992293) [Polish examples/echo_server.rb](https://github.com/socketry/nio4r/commit/7992293db8f069963ebce170fbf6d34e48e9230e)
- (b7d41d5) [CI: Add 2.7 to the matrix](https://github.com/socketry/nio4r/commit/b7d41d5f0288f4f1fb47eca108ca4a8fb8094422)
- (631ca2f) [bump rubocop to 0.74.0](https://github.com/socketry/nio4r/commit/631ca2fb7ae8e8923d949c99bd48eed20274b5f9)
- (22760ae) [Fix Style/ExpandPathArguments](https://github.com/socketry/nio4r/commit/22760ae40761211be3b4cc744c7dbda8f9a9e36f)
- (cd8d391) [Fix Layout/EmptyLineAfterGuardClause](https://github.com/socketry/nio4r/commit/cd8d391da4209cbb683a93b4c374c9297187689d)
- (03b2d2e) [Fix Style/ExpandPathArguments](https://github.com/socketry/nio4r/commit/03b2d2e813e65af9241533a658588b66990a914a)
- (aa67620) [disable Layout/AlignHash](https://github.com/socketry/nio4r/commit/aa676208a9f18712679b0e6fdc5a1b1fae0d34ae)
- (25b3cff) [Fix Lint/UnneededCopEnableDirective](https://github.com/socketry/nio4r/commit/25b3cffca2fb2de1d4370502c2d7582940b8afc0)


</details>